docs/html/development/architecture/index.rst

```diff
@@ -16,10 +16,11 @@ Architecture of pip's internals
 
 
 .. toctree::
-    :maxdepth: 2
+    :maxdepth: 1
```

Comment:
Any particular reason to this change?

Reply:
It looks nicer. I'll make a separate PR.